app/controllers/signup.js

```diff
@@ -63,6 +63,8 @@ export default class SignupController extends Controller {
       errorMessage: 'Username is required',
       required: true,
       showError: false,
+      validator: this.userNameValidator,
+      helpMsg: `Your username should start with your first name having first letter capital. Spaces are are not allowed but hyphens are. Example: If your name is John Doe, then your username can be 'John-doe'`,
```

Comment:
The username should be all lowercase as discussed in the channel. No capital letters allowed.
If your name is `John Doe` and if `john` is not available, you can have `john-doe`